.github/workflows/cd.yaml

```diff
@@ -0,0 +1,15 @@
+# Note: additional setup is required, see https://www.jenkins.io/redirect/continuous-delivery-of-plugins
+
+name: cd
+on:
+  workflow_dispatch:
+#  check_run:
+#    types:
+#      - completed
```

Comment:
```suggestion
  check_run:
    types:
      - completed
```

Reply:
I read that it is mentioned in the [document](https://www.jenkins.io/doc/developer/publishing/releasing-cd/#releasing), if you want to trigger a deployment explicitly, you need to comment out the `check_run` section. Am I misunderstanding here?

Reply:
> Am I misunderstanding here?

Yes. You can trigger a release still, alongside automatic releases. Commenting out is not needed. If you don't want automatic releases, CD is not the right way to go. You should stick to the maven release plugin then.

Reply:
Got it, thanks for your guidance